diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/utils/RulesConfigJsonTest.kt

```diff
@@ -0,0 +1,42 @@
+package org.cqfn.diktat.ruleset.utils
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.cqfn.diktat.common.config.rules.RulesConfig
+import org.cqfn.diktat.common.config.rules.RulesConfigReader
+import org.cqfn.diktat.common.config.rules.getRuleConfig
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.junit.Test
```

Comment:
```suggestion
import org.junit.jupiter.api.Test
```
Please change annotations to junit 5.